packages/cli/src/commands/set-admin.ts

```diff
@@ -151,7 +151,7 @@ function getCommandProps({
       type: 'list',
       choices: [
         {
-          name: 'All proxies',
+          name: 'All instances',
```

Comment:
We need to update the message above as well: `For which proxies would you like to transfer ownership?`